examples/CalcQt/CMakeLists.txt

```diff
@@ -1,36 +1,15 @@
 project(CalcQt)
 
 set(CALCQT_HEADERS src/CalculatorWidget.h)
-set(CALCQT_SOURCES src/CalcQt.cpp src/CalculatorWidget.cpp)
-include_directories(${CUKE_INCLUDE_DIRS} src)
+include_directories(src)
 
-find_package(Qt5Core QUIET)
-find_package(Qt5Widgets QUIET)
-find_package(Qt5Test QUIET)
-
-if(${Qt5Core_FOUND} AND ${Qt5Widgets_FOUND} AND ${Qt5Test_FOUND})
-    set(CMAKE_INCLUDE_CURRENT_DIR ON)
-    set(CMAKE_AUTOMOC ON)
-    set(QT_LIBRARIES Qt5::Core Qt5::Widgets Qt5::Test)
-
-    add_library(libcalcqt src/CalculatorWidget.cpp ${CALCQT_HEADERS})
+if(QT_LIBRARIES)
+    add_library(libcalcqt src/CalculatorWidget.cpp)
```

Comment:
I'm expecting `src/CalculatorWidget.h` to be present here as well. Does automoc work without specifying the header file to run Qt's moc on?

Also please remove the now unused `CALCQT_HEADERS` variable.